tedana/viz.py

```diff
@@ -46,41 +76,47 @@ def write_comp_figs(ts, mask, comptable, mmix, n_vols,
         Reference image to dictate how outputs are saved to disk
 
     """
-
+    # Check that colormap provided exists
+    if png_cmap not in plt.colormaps():
+        LGR.info('Provided colormap is not valid, proceeding with coolwarm')
```

Comment:
I think this should be a warning.

Reply:
does that that feeling about it. changed